src/icons/annotations/IconHighlightText.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import AccessibleSVG from '../accessible-svg';
+import { Icon } from '../iconTypes';
+
+const IconHighlightText = ({
+    className = '',
```

Comment:
Is this needed?

Reply:
Yeah, just following icon files convention. Supporting custom classname is not harmful, right